packages/nextjs/src/config/prefixLoader.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+type LoaderOptions = {
+  distDir: string;
+};
+// TODO Use real webpack types
```

Comment:
Is this gonna be done in a follow-up PR?

Reply:
Probably? Maybe? It's low priority, and slightly complicated by the need to be compatible with both webpack 4 and webpack 5. Not a blocker at the moment.